### PR TITLE
Remove PclZip

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

PclZip should at the very least be moved out of include and into composer, my preference here was to just to remove it entirely in favour of using the built-in PHP zip functions. From what I can see this lib has been marked as deprecated since the initial commit and the composer package hasn't been updated in several years, I can't think of a good reason to bother keeping it.

## How To Test This

1. Test the module loader and upgrade process by installing language packs, custom modules and upgrading the CRM.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by connorshea** [comment]
Conflicts

**Comment by cameronblaikie** [comment]
Assessed :+1:

**Comment by connorshea** [comment]
@Dillon-Brown is there a reason this wasn't backported? I assume Backward Compat?

File: composer.json [hunk 1]
```diff
@@ -55,7 +55,6 @@
     "tedivm/jshrink": "^1.3",
     "tuupola/slim-jwt-auth": "^2.0",
     "wikimedia/composer-merge-plugin": "^1.4",
-    "fullcalendar/fullcalendar": "^3.9",
```
Comment (connorshea) [inline composer.json]: also was this intentional?

**Comment by Dillon-Brown** [comment]
@connorshea Yeah, I think I made most of my "move to composer" PRs to develop for backwards compatibility reasons.

File: composer.json (hunk 1 above)
Comment (Dillon-Brown) [reply composer.json]: Hmm, not sure. I can see that it's still in the develop branch though so it should be fine.

**Pull request merged**